src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -50,6 +50,11 @@
 
 logger = logging.get_logger(__name__)
 
+if not is_torch_greater_or_equal_than_1_10:
+    logger.warning(
+        f"You are using torch=={torch.__version__}, but torch>=1.10.0 is required to use "
+        "MaskFormerModel. Please upgrade torch."
+    )
```

Comment:
We could probably use `torch.tensor(np.pi)` instead of `torch.pi` at https://github.com/huggingface/transformers/blob/c3b42a6f99766e6df051a19f81ca589a2511d71e/src/transformers/models/maskformer/modeling_maskformer.py#L2110

if we want to support as much PT versions as possible.

(But the Past CI for this model have to be re-run in order to get updated results).